{"uid":"e9afe790ba2a9daf","name":"test_valid_login","fullName":"tests.login_test.TestLogin#test_valid_login","historyId":"18657c207d70aa57845e0b7f4751c290","time":{"start":1753434293026,"stop":1753434293029,"duration":3},"description":"Test to verify that a user can log in with valid credentials.","descriptionHtml":"<p>Test to verify that a user can log in with valid credentials.</p>\n","status":"broken","statusMessage":"TypeError: ConfigReader.get_credentials() takes 1 positional argument but 2 were given","statusTrace":"self = <tests.login_test.TestLogin object at 0x105c8e210>\n\n    @allure.story(\"Valid Login\")\n    @allure.description(\"Test to verify that a user can log in with valid credentials.\")\n    def test_valid_login(self):\n        login_page = LoginPage(self.driver)\n>       login_page.login(self.config.get_credentials(\"credentials\"))\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       TypeError: ConfigReader.get_credentials() takes 1 positional argument but 2 were given\n\ntests/login_test.py:12: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1753434290700,"stop":1753434293024,"duration":2324},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test to verify that a user can log in with valid credentials.","status":"broken","statusMessage":"TypeError: ConfigReader.get_credentials() takes 1 positional argument but 2 were given","statusTrace":"self = <tests.login_test.TestLogin object at 0x105c8e210>\n\n    @allure.story(\"Valid Login\")\n    @allure.description(\"Test to verify that a user can log in with valid credentials.\")\n    def test_valid_login(self):\n        login_page = LoginPage(self.driver)\n>       login_page.login(self.config.get_credentials(\"credentials\"))\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nE       TypeError: ConfigReader.get_credentials() takes 1 positional argument but 2 were given\n\ntests/login_test.py:12: TypeError","steps":[],"attachments":[{"uid":"e7ba79c41bd7f50d","name":"stdout","source":"e7ba79c41bd7f50d.txt","type":"text/plain","size":52}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"setup::0","time":{"start":1753434293048,"stop":1753434293124,"duration":76},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Login Tests"},{"name":"story","value":"Valid Login"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"MacBook-Pro-cua-Nguyen-2.local"},{"name":"thread","value":"65088-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"e9afe790ba2a9daf.json","parameterValues":[]}